packages/react-utilities/src/hooks/useOnScrollOutside.ts

```diff
@@ -20,13 +20,13 @@ export const useOnScrollOutside = (options: UseOnClickOrScrollOutsideOptions) =>
 
   React.useEffect(() => {
     if (!disabled) {
-      element?.addEventListener('wheel', listener);
-      element?.addEventListener('touchmove', listener);
+      element?.addEventListener('wheel', listener, !!capture);
```

Comment:
I don't think that it's required for "wheel" and "touchmove". Do you have repro steps for this case?

Reply:
No it's not required, I can revert this one